README.md

```diff
@@ -20,6 +20,8 @@ The LogDNA AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.
     * `LOGDNA_HOSTNAME`: Alternative Host Name *(Optional)*  
     * `LOGDNA_TAGS`: Comma-separated Tags *(Optional)*  
     * `LOGDNA_URL`: Custom Ingestion URL *(Optional)*
+    * `RAW_EVENT_MESSAGE`: Setting `line` to Raw `event.message` *(Optional)*:
+        * You can enable it by setting `RAW_EVENT_MESSAGE` to `YES` or `TRUE`
```

Comment:
I'm not sure I understand the purpose of this line

Reply:
Oh, now I understand. I think this can be reworded to make it easier to understand

Reply:
@jakedipity, what's your suggestion in terms of rewording?